CMakeLists.txt

```diff
@@ -187,6 +187,9 @@ option(USE_SNPE "Use Qualcomm's SNPE library" OFF)
 option(USE_SYSTEM_EIGEN_INSTALL
     "Use system Eigen instead of the one under third_party" OFF)
 option(USE_TENSORRT "Using Nvidia TensorRT library" OFF)
+option(USE_VULKAN "Use Vulkan GPU backend" ON)
+option(USE_VULKANGL "Use VulkanGL GPU backend" OFF)
```

Comment:
Yes, I will rename it.

I thought about organizing it as a fallback option for USE_VULKAN,
something like USE_VULKAN_FALLBACK_GLES to cover logic:  
"try vulkan first, if did not start - try GL approach"

At the moment it uses preprocessor macros so it could not be used like fallback, only as an alternative build time.

Do you think we need 'fallback'? Or at the moment just either USE_VULKAN or USE_GLES with priority if USE_VULKAN  - USE_GLES turned forced to turn off.